rectools/models/sasrec.py

```diff
@@ -406,8 +545,32 @@ def process_dataset_train(self, dataset: Dataset) -> Dataset:
         user_id_map = IdMap.from_values(interactions[Columns.User].values)
         item_id_map = IdMap.from_values(self.item_extra_tokens)
         item_id_map = item_id_map.add_ids(interactions[Columns.Item])
+
+        # get item features
+        item_features = None
+        if dataset.item_features is not None:
+            item_features = dataset.item_features
+            if not isinstance(item_features, SparseFeatures):
```

Comment:
We don't need this check here because we can have item nets that support DenseFeatures and user might want to select them and provide DenseFeatures.

But right now we don''t have time to make the code universal for both Dense and Sparse features. So please write `#TODO: remove assumption on SparseFeatures and add Dense Features support`